### ISSUE TITLE
iOS Standalone app not receiving Push Notifications

### ISSUE THREAD

**Issue by akamuraasai** [expo/expo#2458]
### Environment:
  OS: macOS 10.14
  Node: 10.11.0
  Yarn: 1.10.1
  npm: 6.4.1
  Watchman: 4.7.0
  Xcode: Xcode 10.0 Build version 10A255
  Android Studio: Not Found

### Packages: (wanted => installed)
  expo: ^30.0.0 => 30.0.1
  react: 16.4.2 => 16.4.2
  react-native: ~0.55.2 => 0.55.4

### Diagnostics report:
  https://exp-xde-diagnostics.s3.amazonaws.com/akamuraasai-327d6a2d-cc45-4db1-9477-07d026287ebb.tar.gz

### Target
 iOS Standalone App

### Device
iPhone 8, iOS 12

### Steps to Reproduce

1. Start a new project.
2. Copy and paste do code in Push Notifications Doc.
3. Run expo build:ios
4. Deliver via testflight
5. Send push notification

### Expected Behavior

Receive a push notification and change the text elements with push data.

### Actual Behavior

I'm actually receiving the push if the app is in background, but when I click on it, it just opens the app, but doesn't dispatch the notification event.
In other words, I received that notification, but was _handleNotification not called.

### Reproducible Demo
All the code of the project is the code present in this link of the docs:
[Push Notification Guide](https://docs.expo.io/versions/latest/guides/push-notifications.html#push-notifications)

The only difference is that I'm getting the token via ```getDevicePushTokenAsync```, and sending the push via APNS.

**Comment by ide** [comment]
Have you tested this with Expo push tokens? If it works with Expo push tokens, it means that your push payload is malformed and doesn't conform to the internal protocol (documented by the source code of the Expo client).

**Comment by akamuraasai** [comment]
Hi @ide, reading the source code I've noticed that the **experienceId** key was missing from my push notification test.
Just sending this key with my app **experienceId**, make it works as normal.
You can mark this issue as solved. Thanks.
